### PR TITLE
Multiple custom domains

### PR DESCRIPTION
**Pull request by TheRSP**
This PR closes #

The changes in this PR are as follows:

* Add support for multiple custom domains for a single web app

I have read the [contributing guidelines](CONTRIBUTING.md) and have completed the following:

* [x] **Tested my code** end-to-end against a live Azure subscription.
* [x] **Updated the documentation** in the docs folder for the affected changes.
* [x] **Written unit tests** against the modified code that I have made.
* [x] **Updated the [release notes](RELEASE_NOTES.md)** with a new entry for this PR.
* [x] **Checked the coding standards** outlined in the [contributions guide](CONTRIBUTING.md) and ensured my code adheres to them.

If I haven't completed any of the tasks above, I include the reasons why here:

Below is a minimal example configuration that includes the new features, which can be used to deploy to Azure:

```fsharp
webApp {
  name "my-web-app"
  custom_domain "a.custom-domain.com"
  custom_domain "b.custom-domain.com"
}
```

### CODE REVIEW

File: docs/content/api-overview/resources/web-app.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The Web App builder is used to create Azure App Service accounts. It abstracts t
 | Web App | add_slot | Adds a deployment slot to the app |
 | Web App | add_slots | Adds multiple deployment slots to the app |
 | Web App | health_check_path | Sets the path to your functions health check endpoint, which Azure load balancers will ping to determine which instances are healthy.|
-| Web App | custom_domain | Adds custom domain to the app, containing an app service managed certificate |
+| Web App | custom_domain | Adds a custom domain to the app.  Optionally this can also generate an SSL certificate for the domain. This operator can be used multiple times to add multiple domains |
```
Comment (isaacabraham) [inline docs/content/api-overview/resources/web-app.md:59]: Could you provide an example or simple sentence as to how to generate the SSL cert?

**Review by isaacabraham — APPROVED** [review]
LGTM - if you can just elaborate a little on how to opt-in for SSL generation, that would be great.

Thanks!

File: docs/content/api-overview/resources/web-app.md (hunk 1 above)
Comment (TheRSP) [reply docs/content/api-overview/resources/web-app.md]: I've added a bit more details into the description here. Is this sufficient or should I add an example further down as well?

**Comment by isaacabraham** [comment]
Hey @TheRSP . I've fixed the merge conflicts but don't seem to have permission to push back into this branch. Any chance you can enable that? Then I can push and accept the PR.

Sorry about the delay on this.

**Comment by TheRSP** [comment]
Hi @isaacabraham Thanks for taking a look, I've added you to our repo so you should have write permission now.

**Comment by TheRSP** [comment]
@ninjarobot Merge resolved

**Pull request merged**